src/js/lib/monster.util.js

```diff
@@ -263,6 +263,13 @@ define(function(require) {
 			return isSuperDuper;
 		},
 
+		// Validate if the given port authority ID matches with the current account ID
+		isPortAuthority: function(portAuthorityID, pAccount) {
+			var account = pAccount || (monster.apps.hasOwnProperty('auth') && monster.apps.auth.hasOwnProperty('originalAccount') ? monster.apps.auth.originalAccount : {});
```

Comment:
Once you move back that method in the relevant common control, you should simply check against `self.accountId` since you won't be able to see the related tab while masquerading.

Reply:
seems i don't really needs this method, just compare `_.get(portRequest, 'port_authority', null) === self.accountId` on https://github.com/2600hz/monster-ui/pull/298#discussion-diff-270157809R250